terraform/main.tf

```diff
@@ -69,15 +69,30 @@ module "gh_oidc" {
   }
 }
 
+# 2024-04-18: separate from the others because this was the first one - if we
+# combined the two, this would delete and recreate the service account
 resource "google_service_account" "service_account" {
   account_id   = "rmi-beta-access"
-  display_name = "RMI Beta Access"
+  display_name = "rmi_beta_access"
+}
```

Comment:
Ah ok so this is just renaming an existing service account?

Reply:
Yep! To be more consistent with the other display names.